app/views/reviews/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Enter Your Reviw:</h2>
+<div class="show">
+<%= form_tag "/shelters/#{@shelter.id}/review" do %>
+<%= hidden_field_tag :shelter_id, @shelter.id %>
+
+<%= label_tag :title %>
+<%= text_field_tag :title %><br>
+
+<%= label_tag :rating %>
+<%= text_field_tag :rating %><br>
+
+<%= label_tag :content  %>
+<%= text_field_tag :content %><br>
+
+<%= label_tag :picture %>
+<%= text_field_tag :picture %><br>
+
+<%= label_tag :username %>
+<%= text_field_tag :name_of_user %><br>
+
+<%= label_tag :user_id %>
```

Comment:
Remove this from form. We couldn't find a way to pass in the user_id without using a session, so we added a field.